File: packages/ruby/model/player.rb
class Player
    attr_reader :id
    attr_reader :me
    attr_reader :strategy_crashed
    attr_reader :score
    def initialize(json)
        @id = json["id"]
        @me = json["me"]
        @strategy_crashed = json["strategy_crashed"]
        @score = json["score"]
    end
end

File: packages/javascript/model/Player.js
/**
 * Created by Quake on 18.12.2018
 */
'use strict';

function Player() {

}

Player.prototype.read = function (json) {
    if (typeof json == 'string') {
        json = JSON.parse(json);
    }
    
    this.id = json.id;
    this.me = json.me;
    this.strategy_crashed = json.strategy_crashed;
    this.score = json.score;
}

module.exports = Player;